packages/jest-cli/src/runGlobalHook.js

```diff
@@ -59,7 +59,20 @@ export default ({
       );
 
       // $FlowFixMe
-      const globalModule = require(modulePath);
+      let globalModule = require(modulePath);
+
+      if (
```

Comment:
we should use the helper from `babel` instead of rolling our own: https://github.com/babel/babel/blob/56044c7851d583d498f919e9546caddf8f80a72f/packages/babel-helpers/src/helpers.js#L558-L562

Reply:
Simple change, so I pushed it myself 🙂

Reply:
you can see what I ended up pushing. I think it's fine to expect, if it's an es module, that it's `default`. So we're just handling that, and not special casing named or something for the error message